README.md

```diff
@@ -0,0 +1,96 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Breaking Bad Capstone Project
+
+This is a WEBSITE that displays the information of the Breaking Bad Tv Show with the characters information retrieved from the API and you can leave comments and display likes to your preffered character.
+
+## Built With
+
+Technologies
+
+- Web Pack
+- JavaScript
+- HTML/CSS
+- API
+
+# Live Demo
+
+[Demo](https://erickhansx.github.io/capstone-javascript/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+Our video presentation of the project:
+[loom video](https://www.loom.com/share/eddcfe1a3c9a431d8d4dc96e2a9120d7)
+
+run the following commands in your terminal
+
+- git clone https://github.com/erickhansx/capstone-javascript.git
+
+- git checkout main
+
+### Installation
+
+you can install the following linters into your proejct through node, by these commands:
+
+### - Linters installation:
+
+- Light House: npm install -g @lhci/cli@0.7.x
+- Webhint: npm install --save-dev hint@7.x
+- Stylelint: npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+- ESLint: npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+
+### - Installing Jest, Webpack, others ...
+
+- Node packages: npm install.
+
+### Testing
+
+you can test the web by the following commands:
+
+- npm run test.
+
+### Deployment
+
+you can deploy your project through github pages.
+
+- make a repo for your project on github.
+- go to your project settings.
+- click on pages.
+- select the desired branch to deploy.
+- click save.
+
+## Authors
+
+👤 **Erick Hans Martinez**
+
+- GitHub: [@erickhansx](https://github.com/erickhansx)
+- Twitter: [@TheErickHans](https://twitter.com/TheErickHans)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **LUCCA MICALE**
+
+- GitHub: https://github.com/luccamicale
+- Twitter: https://twitter.com/LuccaMicale_
+- LinkedIn: https://www.linkedin.com/in/lucca-micale-907908233/
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used.
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] As stated by the previous reviewer, kindly make sure to fix license link by fixing the path like this `[MIT](./LICENSE.md)`